forms/collections/elements.js

```diff
@@ -86,7 +93,7 @@ define(function (require) {
     /**
      * Gets a list of element models that have failed validation
      * @param  {Number} fieldLimit The number of fields to return. Defaults to 0
-     * @return {Object} An object of the form { errors: [], total, 19, length: 4}
+     * @return {Object} An object of the form { errors: [field, field, field, ...], total, 19, length: 4}
```

Comment:
Yeah, this isn't required at all. I just figured I'd mention the alternative approach for future improvements.